release.sh

```diff
@@ -3,6 +3,7 @@ npm pack --pack-destination "$PWD" --workspace @mdx-js/language-service
 npm pack --pack-destination "$PWD" --workspace @mdx-js/language-server
 npx --workspace vscode-mdx vsce package --out "$PWD"
 
+node scripts/tag-extension.mjs
```

Comment:
I'd probably tag after creating a successful release (if only you could easily differentiate between that and a failure)

Reply:
I don’t really understand why. Tags are already created succesfully. Also I don’t see this in [`xstate-tools`](https://github.com/statelyai/xstate-tools/blob/8a5df5db0e96e097b51e58280ae6405f2a0f9f94/package.json#L33).

Reply:
Maybe there is some reason why `xstate-tools` does it this way. I honestly don't remember ;p It's just my intuition that a tag should be created *after* releasing and not before it. This is the order in which Changesets does things when it deals with `npm` publishing: https://github.com/changesets/changesets/blob/93ee679537fb43111c29359fe6c82164b359e4d1/packages/cli/src/commands/publish/index.ts#L90-L102

Reply:
Let’s find out then.